codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java

```diff
@@ -159,6 +186,83 @@ public void testResponseDecompression() {
         assertFalse(channel.finish()); // assert that no messages are left in channel
     }
 
+    @Test
+    public void testResponseBrotliDecompression() throws Throwable {
+//        Assume.assumeTrue(Brotli.isAvailable());
+        Brotli.ensureAvailability();
```

Comment:
This needs to get removed and the above line needs to be "uncommented"

Reply:
I beg to differ.
If we manage to make Brotli4j work on all OS tested in Netty's CI, we shouldn't use Assume.
Otherwise, if an update of Brotli4j breaks (eg 1.4.0 was missing the native lib for MacOS), we won't be able to detect it as the test will be automatically silenced.

Reply:
thats actually a very good point.